app/src/main/res/layout/activity_settings.xml

```diff
@@ -346,6 +346,46 @@
                     android:text="@string/display_success_message" />
 
             </RelativeLayout>
+
+            <include
+                android:id="@+id/settings_general_settings_divider"
+                layout="@layout/divider" />
+
+            <TextView
+                android:id="@+id/settings_migrating_label"
```

Comment:
add the label at onResumes text setter arraylist to make sure it uses the proper color